usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -12,16 +12,25 @@
 from usaspending_api.accounts.models import AppropriationAccountBalances, TreasuryAppropriationAccount
 from usaspending_api.awards.models import (
     Award, FinancialAccountsByAwards, FinancialAccountsByAwardsTransactionObligations,
-    TransactionAssistance, TransactionContract, Transaction)
+    TransactionAssistance, TransactionContract, Transaction, AWARD_TYPES, CONTRACT_PRICING_TYPES)
 from usaspending_api.financial_activities.models import FinancialAccountsByProgramActivityObjectClass
 from usaspending_api.references.models import (
     Agency, LegalEntity, Location, RefObjectClassCode, RefCountryCode, RefProgramActivity, CFDAProgram)
 from usaspending_api.submissions.models import SubmissionAttributes
+from usaspending_api.etl.award_helpers import update_awards, update_contract_awards
 
 # This dictionary will hold a map of tas_id -> treasury_account to ensure we don't
 # keep hitting the databroker DB for account data
 TAS_ID_TO_ACCOUNT = {}
 
+# Store some additional support data needed for the type description matching
+award_type_dict = {a[0]: a[1] for a in AWARD_TYPES}
```

Comment:
In my PR from earlier today, I calc the dictionary form of  `AWARD_TYPES` and  `CONTRACT_PRICING_TYPES` in awards/models.py itself, then import those here to save a little code; I'll push a commit shortly to this to re-do that in this PR.